luaclang.c

```diff
@@ -474,7 +524,10 @@ static luaL_Reg cursor_functions[] = {
 static luaL_Reg type_functions[] = {
         {"getSpelling", type_getspelling}, 
         {"getResultType", type_getresult}, 
-        {"getArgType", type_getarg}, 
+        {"getArgType", type_getarg},
+        {"getArrayType", type_getarrtype}, 
```

Comment:
I think we should stick closer to the clang name in this case. Remember to fix the documentation.
```suggestion
        {"getArrayElementType", type_getarrtype}, 
```